packages/components/input-number/src/input-number.vue

```diff
@@ -233,7 +233,7 @@ const setCurrentValue = (
 ) => {
   const oldVal = data.currentValue
   const newVal = verifyValue(value)
-  if (!emitChange) {
+  if (!emitChange && oldVal == newVal) {
```

Comment:
This will cause the change event to be directly triggered every time the value is modified, and the performance will be inconsistent with the original.

Reply:
if (oldVal === newVal && value !== '') return